### PR TITLE
Release 0.1.11 (Alpha)

### PR DESCRIPTION
**Pull request by cwehmeyer**

### CODE REVIEW

**Comment by franknoe** [comment]
Can you also address the update of the conda recipe of 0.1.9 that was 
broken?
(Or maybe remove this version altogether if possible)

Am 25/11/15 um 11:47 schrieb Christoph Wehmeyer:

> Merged #24 https://github.com/markovmodel/thermotools/pull/24.
> 
> —
> Reply to this email directly or view it on GitHub 
> https://github.com/markovmodel/thermotools/pull/24#event-474359058.

## 

---

Prof. Dr. Frank Noe
Head of Computational Molecular Biology group
Freie Universitaet Berlin

Phone: (+49) (0)30 838 75354
Web:   research.franknoe.de

## Mail:  Arnimallee 6, 14195 Berlin, Germany

**Comment by cwehmeyer** [comment]
I would prefer to remove versions 0.1.9 and 0.1.10 as both are broken on either conda or PyPI.

**Comment by franknoe** [comment]
fine for me at this stage. We just shouldn't let broken installation 
recipes linger around.

Am 25/11/15 um 11:55 schrieb Christoph Wehmeyer:

> I would prefer to remove versions 0.1.9 and 0.1.10 as both are broken 
> on either conda or PyPI.
> 
> —
> Reply to this email directly or view it on GitHub 
> https://github.com/markovmodel/thermotools/pull/24#issuecomment-159570993.

## 

---

Prof. Dr. Frank Noe
Head of Computational Molecular Biology group
Freie Universitaet Berlin

Phone: (+49) (0)30 838 75354
Web:   research.franknoe.de

## Mail:  Arnimallee 6, 14195 Berlin, Germany

File: .travis.yml [hunk 1]
```diff
@@ -20,12 +20,12 @@ script:
 after_success:
```
Comment (marscher) [inline .travis.yml:20]: @cwehmeyer why not automatically deploying to PyPI any more?

**Pull request merged**